### ISSUE TITLE
In the post how to add external links？

### ISSUE THREAD

**Issue by vigaea** [klugjo/hexo-theme-magnetic#10]
I have two issues.I really hope that you can help me to solve problems(maybe it`s a problem for me....lol)

1.how to add the pictures' path into the post?Your example are not friendly to me.

2.In the post how to add external links？

**Comment by klugjo** [comment]
1) Can you give me more details on the what picture you are trying to set up ?

2) You can use standard markdown: 

```
[Google](https://www.google.com)
```

**Comment by vigaea** [comment]
Really thanks for your help. My problem is how to add the post's cover_index and cover_detail in each post, but follow your explanation, they always show the  default index page cover image and default_cover_detail. I really confusing(lol....perhaps I too dishes).

**Comment by vigaea** [comment]
for example:https://vigaea.github.io/2017/01/23/2017/

I put the cover_index and cover _detail's external links in the post, is right? Where should I put ?

By the way, Comments didn't work.(soory so many complains, I really love this theme  )

**Comment by klugjo** [comment]
I think your issues are not with my theme but with the fact that you don't fully understand how hexo works. 

Can you paste the `postname.md` file content ? I will help you figure out the right syntax

**Comment by vigaea** [comment]
Here are my post 2017.md content:   (my post address: https://vigaea.github.io/2017/01/23/2017/)
---
title: 2017
---

### Create a new plan

cover_index: "http://ok4no33d8.bkt.clouddn.com/work1.jpg"

cover_detail: "http://ok4no33d8.bkt.clouddn.com/scape.jpg"

**Comment by vigaea** [comment]
Here are my post 2017.md content: (my post address: https://vigaea.github.io/2017/01/23/2017/)


`---
title: 2017
---

### Create a new plan

cover_index: "http://ok4no33d8.bkt.clouddn.com/work1.jpg"

cover_detail: "http://ok4no33d8.bkt.clouddn.com/scape.jpg"`

**Comment by vigaea** [comment]
Here are my post 2017.md content.(how should I modify？)
![20170123185624](https://cloud.githubusercontent.com/assets/12709684/22201433/90396946-e19e-11e6-946c-4582297bf296.png)

**Comment by klugjo** [comment]
Have you been able to solve the problem or can you send me the content of the `.md` file ?

I think you have an error and the content should be something like

```
title: 2017
cover_index: "http://ok4no33d8.bkt.clouddn.com/work1.jpg"
cover_detail: "http://ok4no33d8.bkt.clouddn.com/scape.jpg"
---

### Create a new plan

Here the rest of your content
```

Can you try ?

**Comment by vigaea** [comment]
Yes,I have been solved  all the problem of the post.(lol) Happy new year！

**Comment by vigaea** [comment]
Oh, I'm confused in a problem that is how sort the posts. Should I set up ##date in the post?

**Comment by vigaea** [comment]
Oh, I'm confused in a problem that is how sort the posts. Should I set up `date` in the post?

**Comment by klugjo** [comment]
yes you can use the date to manipulate the order of the posts

**Comment by vigaea** [comment]
As you say, in the home page,the posts show in order,but in archives, the posts are out of order.

**Comment by klugjo** [comment]
That's correct, let me fix that.
